lib/services/osmdata.dart

```diff
@@ -305,10 +307,10 @@ class OsmData{
         < OsmData.getDistance(next, Node(0, latitude, longitude)) ? curr:next);
   }
 
-  Future<List<List<Node>>> calculateRoundTrip(double startLat, double startLong, double distanceInMeter, [int alternativeRouteCount = 1, String poiCategory='']) async{
+  Future<List<HikingRoute>> calculateRoundTrips(double startLat, double startLong, double distanceInMeter, [int alternativeRouteCount = 1, String poiCategory='']) async{
```

Comment:
why not 'calculateHikingRoutes' or 'calculateRoutes'

Reply:
done